chainer/function.py

```diff
@@ -174,6 +174,9 @@ def __call__(self, *inputs):
             return ret[0]
         return ret
 
+    def label(self):
```

Comment:
Make it a property or rename it, since `label` looks like a property or an attribute.